src/modules/rlm_python3/configure.ac

```diff
@@ -84,6 +84,7 @@ if test x$with_[]modname != xno; then
 
 		targetname="rlm_python3"
 	fi
+	AC_CHECK_FUNCS([dl_iterate_phdr])
```

Comment:
Where is this actually used?

Reply:
https://github.com/FreeRADIUS/freeradius-server/blob/4b614008d6f25b4a7ed628d85119ee7db543dd46/src/modules/rlm_python3/rlm_python3.c#L976

Reply:
Need to run `autoheader` too to update the macro definitions.

Reply:
I typically run `autoreconf` which includes autoheader, so that should be done?

Reply:
Just didn't see any changes in this PR to `autoconf.h.in` which there should be with a new function check.

Reply:
If you mean `config.h.in`, that's because it was there already, so this PR hasn't modified that.

Reply:
Weird, OK.  I guess this must have just been omitted in a previous PR.